src/Promenade.Models/Promenade.Models.csproj

```diff
@@ -6,4 +6,9 @@
     <RootNamespace>Ocuda.Promenade.Models</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.Mvc" Version="5.2.7" />
```

Comment:
Why are we including `Microsoft.AspNet.Mvc` in a model project? Can this be removed?

Reply:
If SelectList is needed in the model it's better to include `Microsoft.AspNetCore.Mvc.ViewFeatures` instead.

Reply:
We havent used SelectList so far

Reply:
We havent used SelectList so far. Unsure how it was added

Reply:
If we aren't using anything in the package then let's remove the package from the project.